python/setup.py

```diff
@@ -167,7 +167,6 @@ def _supports_symlinks():
                   'pyspark.ml',
                   'pyspark.ml.linalg',
                   'pyspark.ml.param',
-                  'pyspark.ml.stat',
```

Comment:
@holdenk @viirya, quick question here: I see that this `pyspark.ml.stat` line was added in #16465. The problem with this line is that it seems to cause failures during `python setup.py sdist` packaging because that process searches for a non-existent `pyspark/ml/stat` file / directory.

Was this `ml.stat` just a typo?